### DISCUSSION TITLE
新手求教如何生成element-plus的离线文档

### DISCUSSION THREAD

**Discussion by jiachxxj** [element-plus/element-plus#3458] [Q&A]
在网上查不到相关说明，对于小白来讲，不知道该运行什么命令来构建文档。
因为项目是在离线环境下的，想构建离线文档方便查阅。
求教！跪谢！

**Comment by Alanscut** [comment]
clone 源码之后，先安装依赖，然后运行 `npm run website-dev` 或 `yarn website-dev`，默认 `localhost:8080` 即可访问离线文档了

**Reply by jiachxxj** [reply]
> clone 源码之后，先安装依赖，然后运行 `npm run website-dev` 或 `yarn website-dev`，默认 `localhost:8080` 即可访问离线文档了

可是在package.json中没有website-dev这个命令啊，运行之后也是提示命令不存在

**Comment by sxzz** [comment]
```bash
git pull
yarn install
cd docs && yarn install && cd -
yarn docs:gen-locale
yarn docs:build # For build docs
# yarn docs:dev # For development
```

**Reply by jiachxxj** [reply]
我看这个package.json里面的命令，是不是在windows系统下执行不了啊

**Reply by sxzz** [reply]
目前似乎是的，尚未考虑 Windows 系统。

**Comment by slg33856023** [comment]
借楼提问：打包文档项目后，必须要部署吗？

**Reply by slg33856023** [reply]
我是想直接打开index.html文件就能看文档，不喜欢折腾部署。
